disnake/threads.py

```diff
@@ -182,6 +183,7 @@ def _from_data(self, data: ThreadPayload):
         self.slowmode_delay = data.get("rate_limit_per_user", 0)
         self.message_count = data.get("message_count")
         self.member_count = data.get("member_count")
+        self._newly_created: Optional[bool] = data.get("newly_created")
```

Comment:
Defaults to `False`?

Reply:
I don't think it's necessary, it's a private attribute anyway.